nixos/modules/services/audio/mopidy.nix

```diff
@@ -99,6 +95,7 @@ in {
       extraGroups = [ "audio" ];
       description = "Mopidy daemon user";
       home = "${cfg.dataDir}";
+      createHome = true;
```

Comment:
in my experience `createHome` is less reliable. It's executed only once and not directly tied to the service startup.

I've had cases where the home had been created, a filesystem mounted on top of it afterwards, and finally the service starting and breaking.

Reply:
Great point.